SpeechResponder/eddi.json

```diff
@@ -1869,7 +1869,7 @@
       "enabled": true,
       "priority": 3,
       "responder": true,
-      "script": "{_ Ship targeted }\r\n{_ Triggered when the player selects a target }\r\n\r\n{_ event Parameters:   targetlocked - bool (ie false when losing target) }\r\n{_ If target locked:   name (of ship), scanstage (ie 0, 1, 2, 3) }\r\n{_ If Scan stage >= 1: name (of pilot), rank (of pilot) }\r\n{_ If scan stage >= 2: shieldhealth, hullhealth (both 0-100) }\r\n{_ If scan stage >= 3: faction, legalstatus (ie clean, wanted, etc), bounty (if appicable), }\r\n{_                     subsystem (ie FSD, power plant, etc), subsystemhealth (0-100) }\r\n\r\n\r\n{_ Context }\r\n{SetState('eddi_context_last_subject', 'ship')}\r\n{SetState('eddi_context_last_action', 'targeted')}\r\n\r\n{if event.targetlocked && event.scanstage > 2 && !event.subsystem:\r\n    {set legalstatus to event.legalstatus}\r\n    {if legalstatus = \"Wanted\" || legalstatus = \"Wanted Enemy\" || legalstatus = \"Warrant\":\r\n        {set wanted_list to split(state.eddi_context_wanted_list, \"+=\")}\r\n        {if find(wanted_list, event.name) < 0:\r\n            {event.name} {OneOf(\"has a bounty\", \"is fair game\", \"is a legitimate target\")}.\r\n            {if len(wanted_list) = 0: {set sep to \"\"} |else: {set sep to \"+=\"}}\r\n            {set wanted_list to cat(wanted_list, [event.name])}\r\n            {SetState('eddi_context_wanted_list', join(wanted_list, sep))}\r\n        }    \r\n    }\r\n}\r\n\r\n",
+      "script": "{_ Ship targeted }\r\n{_ Triggered when the player selects a target }\r\n\r\n{_ event Parameters:   targetlocked - bool (ie false when losing target) _}\r\n{_ If target locked:   name (of ship), scanstage (ie 0, 1, 2, 3) _}\r\n{_ If Scan stage >= 1: name (of pilot), rank (of pilot) _}\r\n{_ If scan stage >= 2: shieldhealth, hullhealth (both 0-100) _}\r\n{_ If scan stage >= 3: faction, power (if player is pledged), _}\r\n{_                     legalstatus (ie clean, wanted, etc), bounty (if appicable), _}\r\n{_                     subsystem (ie FSD, power plant, etc), subsystemhealth (0-100) _}\r\n\r\n\r\n{_ Context }\r\n{SetState('eddi_context_last_subject', 'ship')}\r\n{SetState('eddi_context_last_action', 'targeted')}\r\n\r\n{if event.targetlocked && event.scanstage > 2 && !event.subsystem:\r\n    {set legalstatus to event.legalstatus}\r\n    {if legalstatus = \"Wanted\" || legalstatus = \"Wanted Enemy\" || legalstatus = \"Warrant\":\r\n        {set wanted_list to split(state.eddi_context_wanted_list, \"+=\")}\r\n        {if find(wanted_list, event.name) < 0:\r\n            {event.name} {OneOf(\"has a bounty\", \"is fair game\", \"is a legitimate target\")}.\r\n            {if len(wanted_list) = 0: {set sep to \"\"} |else: {set sep to \"+=\"}}\r\n            {set wanted_list to cat(wanted_list, [event.name])}\r\n            {SetState('eddi_context_wanted_list', join(wanted_list, sep))}\r\n        }    \r\n    }\r\n}\r\n",
```

Comment:
Slight typo... "appicable" should be "app**l**icable"

Reply:
Power is also only written when both the player and target are aligned to a power.

Reply:
Lol, that 'appicable' error has been there for awhile. Good catch. Fixed.

The target being aligned is obviously implicit. If the player is aligned, then the result will be the target power alignment, or otherwise `None`.